jupiterone/data_source_user_group.go

```diff
@@ -0,0 +1,169 @@
+package jupiterone
```

Comment:
nit: choose a more descriptive package name.  Importing `jupiterone` doesn't necessarily tell me much about what the package does at a glance